### ISSUE TITLE
more description about the warmup behaviour

### ISSUE THREAD

**Issue by fransiskusyoga** [LEFTeyex/warmup#2]
I want to ask about the warmup behavior. For example, I use StepLR with step_size=5, gamma=0.1, start_lr=0.3, len(dataloader)=100. If I set the warmup_step=800, what is the learning rate after warmup_step? is it 0.3 or 0.03?
